yb-voyager/cmd/exportData_test.go

```diff
@@ -94,6 +121,39 @@ func setupPostgreDBAndExportDependencies(t *testing.T, sqls []string, schemas st
 	return testExportDir
 }
 
+func setupYugabyteDBSourceAndExportDependencies(t *testing.T, sqls []string, schemas string) string {
```

Comment:
you're basically setting up postgres and yugabytedb both, right? Perhaps call it `setupPostgresSourceAndYugabyteTarget` or something similar?